composer.json

```diff
@@ -11,7 +11,7 @@
     "require": {
         "php": ">=5.4.0",
         "league/flysystem": "~1.0",
-        "aws/aws-sdk-php": "3.*@dev"
+        "aws/aws-sdk-php": "^3.0.0"
```

Comment:
More modern, and it's the default now when you do `composer require aws/aws-sdk-php`. And if jeremeamia would push a critical security fix or something then we would only have to update `^3.0.0` to `^3.0.x`.